scss/bootstrap.scss

```diff
@@ -12,6 +12,7 @@
 @import "variables";
 @import "mixins";
 @import "utilities";
+@import "./mixins/form-builder-mixins";
```

Comment:
Let's move this to import to `_mixins.scss` so that it is loaded with the others using the same approach.